multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -0,0 +1,124 @@
+# This butane config will do the following:
+#
+# - Merge in the builder-common.ign Ignition file
+# - Allow the builder user to log in with the associated ssh key
+# - Set a hostname
+# - Create and initialize the secex-data volume
+#
+variant: fcos
+version: 1.4.0
+ignition:
+  config:
+    merge:
+      - source: http://foo/bar/builder-common.ign
+passwd:
```

Comment:
this passwd section is mostly duplicated with what's in `builder-common.bu` Maybe let's just add your SSH key to `builder-common.bu` and remove this?

Reply:
Yes, i think this makes it easier to maintain and also makes `coreos-s390x-rhcos-builder.bu` simply a variant of `coreos-s390x-builder.bu`.
I updated the PR accordingly.